### DISCUSSION TITLE
Question regarding contributing to Cockpit

### DISCUSSION THREAD

**Discussion by Baizkunthaz** [cockpit-project/cockpit#19752] [General]
<h3>Hey, sorry to interrupt. I hope you guys are doing well. I am not only new to this org but also to open source, so I need some guidance and help regarding contributing to this organization; I would be grateful if you would provide me with some advice. THANKSSSSSS </h3>

**Comment by subhoghoshX** [comment]
Hey @Baizkunthaz welcome :) Please follow https://github.com/cockpit-project/cockpit/blob/main/HACKING.md to set up a development environment. Try changing some string in the code to see it reflect in the browser.

After that you may browser the [good first issues](https://github.com/cockpit-project/cockpit/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue) to find something you're interested in.

You can also join the [Matrix](https://matrix.to/#/#cockpit:fedoraproject.org) channel to ask questions. Thanks!

**Reply by Baizkunthaz** [reply]
Should I need to install Linux to contribute to the cockpit?

**Reply by subhoghoshX** [reply]
Yes, you need to install Linux. I heard some people trying it with WSL but I don't think anyone in the team uses it.
